primeui-tester/src/main/java/com/axonivy/ivy/webtest/primeui/AjaxHelper.java

```diff
@@ -13,7 +13,7 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-package com.axonivy.ivy.supplements.primeui.tester;
+package com.axonivy.ivy.webtest.primeui;
```

Comment:
know many projects that relied upon this will face compiler errors as soon as they update to the latest version.
but still i think it's a good idea to make this change know -> just ensure that our own consumer of these artifacts will all rely upon the latest version ....and fix the imports where needed :)

Reply:
Do you think this project is used by some customer projects? 
May add a changelog note?